src/UserSpaceInstrumentation/Attach.cpp

```diff
@@ -122,8 +122,13 @@ ErrorMessageOr<void> DetachAndContinueProcess(pid_t pid) {
   auto tids = GetTidsOfProcess(pid);
   for (auto tid : tids) {
     if (ptrace(PTRACE_DETACH, tid, nullptr, nullptr) == -1) {
-      return ErrorMessage(
-          absl::StrFormat("Error while detaching from thread %d: %s", tid, SafeStrerror(errno)));
+      // Failing with "no such process" is fine here: The thread might have been created (in running
+      // state) while we were attached.
```

Comment:
I would clarify that as a result we never attached to that thread. It's only clear from the commit message.

Reply:
done